products/nodejs.md

```diff
@@ -26,6 +26,14 @@ auto:
   -   git: https://github.com/nodejs/node.git
 
 releases:
+-   releaseCycle: "22"
+    releaseDate: 2024-04-23
```

Comment:
```suggestion
    releaseDate: 2024-04-24
```

See https://nodejs.org/en/blog/announcements/v22-release-announce and https://github.com/nodejs/node/releases/tag/v22.0.0.